modules/nextflow/src/main/groovy/nextflow/executor/BashWrapperBuilder.groovy

```diff
@@ -427,6 +428,8 @@ class BashWrapperBuilder {
         /*
          * create a builder instance given the container engine
          */
+        if( engine == 'charliecloud' )
+            return new CharliecloudBuilder(containerImage)
         if( engine == 'docker' )
```

Comment:
One minor thing, let's move this as the last. Sorry, last arrived, last served 😄

Reply:
The section looked almost alphabetical :sweat_smile: 
I changed it